packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3933,38 +3971,4 @@ void main() {
       TargetPlatform.windows,
     }),
   );
-
-  testWidgets('The handles show after pressing Select All (iOS and Mac)', (WidgetTester tester) async {
```

Comment:
I believe there is now no way to make Select All appear for a SelectableText on iOS, so I've removed this test.

Reply:
That seems consistent with native iOS in my experimentation.